README.md

```diff
@@ -178,6 +178,11 @@ All of these libraries are already bundled within the Infusion image.
 
 Infusion is in the process of switching to use [Stylus](http://learnboost.github.io/stylus/) for CSS pre-processing.
 CSS files for the Preferences Framework have been re-written in Stylus. Only Stylus files are pushed into the github repository.
+
 For developing the Preferences Framework, run the following from the project root to compile Stylus files to CSS:
 
-    grunt buildStylus
+    `grunt buildStylus`
```

Comment:
The indentation had already put this in a code block. Perhaps you want to change it to the explicit code block with  "```"